notesapi/v1/models.py

```diff
@@ -0,0 +1,145 @@
+import json
+from django.db import models
+from django.core.exceptions import ValidationError
+from django.conf import settings
+from django.db.models import signals
+from django.dispatch import receiver
+from elasticutils.contrib.django import Indexable, MappingType
+
+
+
+class Note(models.Model):
+    """
+    Annotation model.
+    """
+    # no indexes here because retrieval is taken care of by ElasticSearch
```

Comment:
> related to my comment about List and Detail get methods, I do think it would make sense to add database indexes for the most commonly anticipated access patterns - user_id + course_id is mainly what I'm thinking.

Added.